docs/source/catalog.rst

```diff
@@ -381,6 +381,28 @@ For cases where the catalog is not defined in a YAML spec, the argument ``user_p
 to the constructor takes the same form as ``parameters`` above: a dict of user parameters,
 either as ``UserParameter`` instances or as a dictionary spec for each one.
 
+Templating parameters
+''''''''''''''''''
+
+Template functions can also be used in parameters (see `Templating`_, above), but you can use the available functions directly without the extra `{{...}}`.
+
+For example, this catalog entry uses the ``env("USER")`` functionality as described to set a default based on the user's home directory.
```

Comment:
You meant HOME here, right? I don't know offhand whether this works on windows...

Reply:
On reflection, I think it can be left as is, and maybe it does indeed work on windows. We are generally interfacing with fsspec anyway, which casts paths to posix-like, even on windows.